samples/python/appengine-flexible/README.md

```diff
@@ -0,0 +1,76 @@
+# Snapshot Debugger Examples for Python in the App Engine flexible environment
+
+NOTE: This sample application was copied from
+[python-docs-samples/appengine/hello-world/flexible][sample-source]
+and modified for the Snapshot Debugger samples here.
+
+
+This is the sample application for the
+[Quickstart for Python in the App Engine flexible environment](https://cloud.google.com/appengine/docs/flexible/python/quickstart)
+tutorial found in the [Google App Engine Python flexible environment](https://cloud.google.com/appengine/docs/flexible/python)
+documentation.
+
+* [Setup](#setup)
+* [Deploying to App Engine](#deploying-to-app-engine)
+
+## Setup
+
+Before you can run or deploy the sample, you need to do the following:
+
+1.  Refer to the [Snapshot Debugger readme](../../../README.md) file for
+    instructions on setting up the snapshot debugger.
+1.  Refer to the [getting started documentation][create-cloud-project]
+    for instructions on setting up your Cloud project.
+
+## Deploying to App Engine
+
+The following code changes have been made to enable the Snapshot Debugger:
+
+https://github.com/GoogleCloudPlatform/snapshot-debugger/blob/ae8358fb69ae8f9df10e4675d59671d950e0d6c1/samples/python/appengine-flexible/requirements.txt#L4
+
+https://github.com/GoogleCloudPlatform/snapshot-debugger/blob/ae8358fb69ae8f9df10e4675d59671d950e0d6c1/samples/python/appengine-flexible/main.py#L17-L21
+
+Then deploy to App Engine as usual:
+
+    gcloud app deploy
+
+## Navigate To Your App
+
+This will ensure the app is run and is required as your app will not be
```

Comment:
In flex this step isn't actually necessary, it's in standard that the app needs to be started for  the first request. That said to keep the instructions the same between them it's probably fine.

Reply:
Good point; I'll remove it.